docs: Move N4A FAQ content to docs (#537)
nginx/documentation: content/nginxaas-azure/getting-started/prerequisites.md

- how-to
---

> ## NGINX Plus users

Reviewer: This section needs more than just certificates. An NGINX Plus user could be anywhere, another cloud, onprem, cloud specific tech and certificates is just one part of it. It could also be that the customer does not have SSL as they don't care about it.

Other things that a user may have to do in order to migrate:
- Review their configuration and make necessary changes.

- Based on what the customer is wanting to do, they may have to make changes on setting up Azure networking to route to their upstreams or also move their upstreams into Azure for stuff to work inside of Azure.
- They may have to deal with DNS as well.

The above is not an exhaustive list and there could be more scenarios where a user will have to do more than just `certificates`.

Author: The content here is the content that was in the FAQ. There wasn't more content in the FAQ. Adding content is outside of the scope of this PR. We can either:

- Move this content somewhere else.
- Keep the content as is in this PR and work separately on adding the missing part after the PR is merged.
- Delete the content.

Reviewer: @russokj I am in the favor of dropping this text as it is incomplete. At a minimum, we need to mention something about the user NGINX configuration here:

_Get a hold of your NGINX configuration so it can be applied to the deployment. **NOTE:** If you have additional files that are needed for NGINX to function holistically, please have those handy as they will have to be applied to the deployment._

We can phrase the above nicely if we want to do this here and move on, or we drop it, file the work, and revisit it.

Author: removing with section for now to unblock the PR

If you are an existing F5 NGINX Plus user and you want to migrate to F5 NGINX as a Service for Azure (NGINXaaS), you need to know the following:

> - In NGINX Plus, you SSH into the NGINX Plus system, store your certificates in some kind of storage and configure the network and subnet to connect to NGINX Plus.

Reviewer 2: seems redundant

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates and configure the network and subnet to connect to NGINX Plus.
```

Author: need to emphasize that the certificates are kept in a file storage system (as opposed to Azure Key Vault)

@mjang please suggest again

Reviewer 2: Oh, one of those filesystem things.... maybe? 

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates locally, and configure the network and subnet to connect to NGINX Plus.
```

Reviewer 2: or maybe

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates on that system, and configure the network and subnet to connect to NGINX Plus.
```

Reviewer 2: Or maybe -- if you consider NFS drives

```suggestion
- In NGINX Plus, you SSH into the NGINX Plus system, store your certificates in the filesystem, and configure the network and subnet to connect to NGINX Plus.
```

Author: I favor "file system"
